nodes/transforms/noisize.py

```diff
@@ -0,0 +1,192 @@
+
+def vector_noise_multi_seed(params, noise_basis='PERLIN_ORIGINAL'):
+    vert, seed_val, scale_out, matrix = params
+    noise.seed_set(seed_val if seed_val else 1385)
+    v_vert = Vector(vert)
+    noise_v = noise.noise_vector(matrix @ v_vert, noise_basis=noise_basis)[:]
+    return v_vert + Vector((noise_v[0]*scale_out[0], noise_v[1]*scale_out[1], noise_v[2]*scale_out[2]))
+
+def vector_noise_normal(params, noise_basis='PERLIN_ORIGINAL'):
+    vert, scale_out, matrix, normal = params
+    v_vert = Vector(vert)
+    scalar_noise = deepnoise(matrix @ v_vert, noise_basis=noise_basis)
+    noise_v = Vector(normal) * scalar_noise
+    return v_vert + Vector((noise_v[0]*scale_out[0], noise_v[1]*scale_out[1], noise_v[2]*scale_out[2]))
+
+def vector_noise_normal_multi_seed(params, noise_basis='PERLIN_ORIGINAL'):
+    vert, seed_val, scale_out, matrix, normal = params
+    v_vert = Vector(vert)
+    noise.seed_set(seed_val if seed_val else 1385)
+    noise_scalar = deepnoise(matrix @ v_vert, noise_basis=noise_basis)
+    noise_v = Vector(normal) *noise_scalar
+    return v_vert + Vector((noise_v[0]*scale_out[0], noise_v[1]*scale_out[1], noise_v[2]*scale_out[2]))
+
+def v_noise(verts, pols, m_prop, noise_type, result):
+    func = vector_noise if len(m_prop) == 2 else vector_noise_multi_seed
+    result.append([func(v_prop, noise_basis=noise_type) for v_prop in zip(verts, *m_prop)])
+
+def v_normal(verts, pols, m_prop, noise_type, result):
+    bm = bmesh_from_pydata(verts, [], pols, normal_update=True)
+    normals = [Vector(v.normal) for v in bm.verts]
+    func = vector_noise_normal if len(m_prop) == 2 else vector_noise_normal_multi_seed
+    result.append([func(v_prop, noise_basis=noise_type) for v_prop in zip(verts, *m_prop, normals)])
+    bm.free()
+
+def noisize(params, constant, matching_f):
+    result = []
+    noise_function, noise_type, match_mode = constant
+    params = matching_f(params)
+    local_match = iter_list_match_func[match_mode]
+    for props in zip(*params):
+        verts, pols, seed_val, scale_out, matrix = props
+        if type(matrix) == list:
+            matrix = [m.inverted() for m in matrix]
+        else:
+            matrix = [matrix.inverted()]
+        if len(seed_val) > 1:
+            m_prop = local_match([seed_val, scale_out, matrix])
+        else:
+            m_prop = local_match([scale_out, matrix])
+            seed_val = seed_val[0]
+            noise.seed_set(int(seed_val) if seed_val else 1385)
+        noise_function(verts, pols, m_prop, noise_type, result)
+
+    return Vector_degenerate(result)
+
+avail_noise = [(t[0], t[0].title(), t[0].title(), '', t[1]) for t in noise_options]
+noise_func = {'NORMAL': v_normal, 'VECTOR': v_noise}
+
+class SvNoisizeNode(bpy.types.Node, SverchCustomTreeNode):
+    """
+    Triggers: Noise Displacement
+    Tooltip: Move input verts/mesh with a noise values.
+
+    """
+
+    bl_idname = 'SvNoisizeNode'
+    bl_label = 'Noisize'
```

Comment:
this name / label  ,  could be better... "Noise Displacer" even.  Not a big issue, maybe Noisize feels a little gimmicky ?

Reply:
It was recalling Randomize but it is true Noisize  is a little weird...

Reply:
"Perturb", "Add Noise"

Reply:
Noise Displacer sounds fine to me, but would you change also the class name and the files names?

Reply:
Maybe just "Noise Displace" is even cleaner

Reply:
yeah, but do it early.